### ISSUE TITLE
Does argument 'censored' make sense for our classif / regr NNet learners?

### ISSUE THREAD

**Issue by mb706** [mlr-org/mlr3learners#313]
NNet doc about the 'censored' hyperparameter says

> A variant on softmax, in which non-zero targets mean possible classes. Thus for `softmax` a row of `(0, 1, 1)` means one example each of classes 2 and 3, but for `censored` it means one example whose class is only known to be 2 or 3.

This sounds like it is about multilabel tasks, but our tasks always have a single label for each sample, so this should not apply, right? Also our regression NNet should not have this hyperparameter at all? I haven't read the docs thoroughly, though, so not sure.
